src/Activation/AcceptEula.tsx

```diff
@@ -83,78 +56,127 @@ const AcceptEula: FunctionComponent = () => {
-          label={t("common.continue")}
+    <>
+      <GradientBackground />
+      <SafeAreaView style={style.container}>
+        <GlobalText style={style.headerText}>
+          {t("onboarding.terms_header_title")}
+        </GlobalText>
+        <EulaLink
+          docName={t("onboarding.privacy_policy")}
+          onPress={linkToPrivacyPolicy}
         />
-      </View>
-    </SafeAreaView>
+        <EulaLink docName={t("onboarding.eula")} onPress={linkToEula} />
+        <View style={style.footerContainer}>
+          <TouchableOpacity
+            style={style.checkboxContainer}
+            onPress={() => toggleCheckbox(!boxChecked)}
+            accessible
+            accessibilityRole="checkbox"
+            accessibilityLabel={checkboxLabel}
+            testID="accept-terms-of-use-checkbox"
+          >
+            <SvgXml
+              xml={checkboxIcon}
+              fill={Colors.primary100}
+              width={Iconography.small}
+              height={Iconography.small}
+            />
+            <GlobalText style={style.checkboxText}>
+              {t("onboarding.eula_agree_terms_of_use")}
+            </GlobalText>
+          </TouchableOpacity>
+          <Button
+            onPress={handleOnPressNext}
+            disabled={!boxChecked}
+            label={t("common.continue")}
+          />
+        </View>
+      </SafeAreaView>
+    </>
   )
 }
 
+type EulaLinkProps = {
+  docName: string
+  onPress: () => Promise<void>
+}
+const EulaLink: FunctionComponent<EulaLinkProps> = ({ docName, onPress }) => {
+  const { t } = useTranslation()
+  return (
+    <TouchableOpacity style={style.eulaLinkContainer} onPress={onPress}>
+      <View style={style.eulaTextContainer}>
+        <GlobalText style={style.eulaText}>
+          {t("onboarding.please_read_the")}
+        </GlobalText>
+        <GlobalText style={{ ...style.eulaText, ...style.eulaLink }}>
+          <> {docName}</>
+        </GlobalText>
+      </View>
+      <SvgXml
+        xml={arrow}
+        fill={Colors.primary100}
+        style={style.eulaLinkArrow}
+      />
+    </TouchableOpacity>
+  )
+}
 const style = StyleSheet.create({
   container: {
+    ...Layout.positionOverBackground,
     flex: 1,
+    margin: Spacing.xxLarge,
+  },
+  headerText: {
+    ...Typography.header1,
+  },
+  eulaLinkContainer: {
+    flexDirection: "row",
+    alignItems: "center",
+    justifyContent: "space-between",
+    padding: Spacing.small,
+    marginTop: Spacing.large,
     backgroundColor: Colors.primaryLightBackground,
-    height: "100%",
+    ...Outlines.roundedBorder,
+    ...Outlines.baseShadow,
+    shadowColor: Colors.neutral30,
+    shadowRadius: 5,
+    borderColor: Colors.neutral25,
   },
-  loadingIndicator: {
-    justifyContent: "center",
-    height: "100%",
+  eulaTextContainer: {
+    flexDirection: "row",
+    alignItems: "center",
+    flex: 1,
+    flexWrap: "wrap",
+  },
+  eulaText: {
+    ...Typography.largeFont,
```

Comment:
Is it possible to use `Typography.body1` and add a larger size here? Or perhaps one of the header vars?